gateway/src/test/java/io/camunda/zeebe/gateway/api/job/LongPollingActivateJobsTest.java

```diff
@@ -672,6 +674,29 @@ public BrokerResponse<?> handle(final BrokerActivateJobsRequest request)
     assertThat(request.hasScheduledTimer()).isFalse();
   }
 
+  @RegressionTest("https://github.com/camunda/zeebe/issues/13867")
+  @Test
+  public void shouldExecuteOnCancelRequestHandlerOnActor() {
```

Comment:
:X: The CI reports this test is flaky. I'm not sure why it's failing.

```
Error:  io.camunda.zeebe.gateway.api.job.LongPollingActivateJobsTest.shouldExecuteOnCancelRequestHandlerOnActor -- Time elapsed: 1.963 s <<< ERROR!
java.lang.NullPointerException: Cannot invoke "io.camunda.zeebe.gateway.impl.job.LongPollingActivateJobsHandler.accept(io.camunda.zeebe.scheduler.ActorControl)" because "this.handler" is null
	at io.camunda.zeebe.gateway.api.job.LongPollingActivateJobsTest.shouldExecuteOnCancelRequestHandlerOnActor(LongPollingActivateJobsTest.java:691)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

Reply:
tbh I would remove it anyway, like I wrote, I don't think it's a very useful test, but I couldn't come up with anything else for this specific concurrency edge case.